app/views/devise/sessions/new.html.erb

```diff
@@ -34,14 +34,14 @@
       ) %>
   <%= f.input :request_id, as: :hidden, input_html: { value: @request_id } %>
   <div class='margin-bottom-4'>
-    <%= f.submit t('links.next'), full_width: true, big: false, wide: false %>
+    <%= f.submit t('links.next'), full_width: true, big: true, wide: false %>
     <h2 class='separator-text'>
       <%= t('headings.create_account_with_sp.cta', app_name: APP_NAME) %>
     </h2>
     <%= link_to(
           t('links.create_account'),
           sign_up_email_url(request_id: @request_id),
-          class: 'text-no-underline usa-button usa-button--outline usa-button--full-width margin-top-1',
+          class: 'text-no-underline usa-button usa-button--big usa-button--outline usa-button--full-width margin-top-1',
```

Comment:
Minor / unrelated: It occurs to me that the `text-no-underline` class is redundant with `usa-button` which already defines those styles. We could remove it.

```suggestion
          class: 'usa-button usa-button--big usa-button--outline usa-button--full-width margin-top-1',
```